src/jdk.sctp/unix/classes/sun/nio/ch/sctp/SctpMultiChannelImpl.java

```diff
@@ -30,12 +30,8 @@
 import java.net.InetSocketAddress;
 import java.io.FileDescriptor;
 import java.io.IOException;
-import java.util.Collections;
+import java.util.*;
```

Comment:
It would be better to avoid wildcards in regular imports.

Reply:
Agreed. I would have to find the settings in my IDE to prevent this automatic "feature".